src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin circle($circle-size) {
+  height: $circle-size;
+  width: $circle-size;
+  border-radius: 50%;
+}
+
+@mixin center($center-size) {
```

Comment:
u pass argument but never use it.. do we really need it?

Reply:
no, i just tried it before and forgot deleted it

Reply:
hmm, sry but i use it all in this watches

Reply:
![image](https://user-images.githubusercontent.com/110053436/218751744-32327b36-57dd-4b7d-a3d8-cc8622fbe224.png)
i dont know which argument you mean, but if you talking about border-radius - i can say what that argument help me to hide red center of bg-image

Reply:
I'm talking about the $center-size parameter